backend/question-service/src/types/CreateQuestionDto.ts

```diff
@@ -23,18 +23,36 @@ export class CreateQuestionDto {
     @IsUrl()
     link: string
 
-    constructor(title: string, description: string, categories: Category[], complexity: Complexity, link: string) {
+    @IsArray()
+    @IsString({ each: true })
+    testInputs: string[]
```

Comment:
im assuming that complex test inputs like nested arrays will be JSON.stringify ied

Reply:
does judge0 automatically parse it or do we need to parse it ourselves before sending it out?

Reply:
nope, current implementation is such that the inputs and expected outputs should be in string, user has the responsibility to ensure parsing and formatting is done properly